<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Regels</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QScrollArea" name="scrollArea">
          <property name="styleSheet">
           <string notr="true">#listSelector{ background-color: black; }</string>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>700</width>
             <height>729</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QDockWidget" name="dwFunction">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>274</width>
             <height>141</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>274</width>
             <height>117</height>
            </size>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Functies</string>
           </property>
           <widget class="QWidget" name="dockWidgetContents_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="ElementView" name="vwFunction"/>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dwPredef">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>590</y>
             <width>274</width>
             <height>141</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>224</width>
             <height>113</height>
            </size>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Voorgedefinieerde elementen</string>
           </property>
           <widget class="QWidget" name="dockWidgetContents_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="PredefinedElementView" name="vwPredef">
               <property name="acceptDrops">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dwAction">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>140</y>
             <width>274</width>
             <height>141</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>117</height>
            </size>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Acties</string>
           </property>
           <widget class="QWidget" name="dockWidgetContents_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="ElementView" name="vwAction"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dwConstant">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>274</width>
             <height>151</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>207</width>
             <height>151</height>
            </size>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Constanten</string>
           </property>
           <widget class="QWidget" name="dockWidgetContents_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="ElementView" name="vwConstant"/>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QDockWidget" name="dwFeature">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>440</y>
             <width>274</width>
             <height>141</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>112</width>
             <height>111</height>
            </size>
           </property>
           <property name="features">
            <set>QDockWidget::DockWidgetMovable</set>
           </property>
           <property name="windowTitle">
            <string>Eigenschappen</string>
           </property>
           <widget class="QWidget" name="dockWidgetContents_11">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="ElementView" name="vwFeature"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Gemeenschappelijke tafel</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWebView" name="wvCommon">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Testtafel</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWebView" name="wvOwn">
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEdit_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Code"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionShow_information"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExport"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;new</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>Sluiten</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionTo_Prolog">
   <property name="text">
    <string>Naar &amp;Prolog</string>
   </property>
  </action>
  <action name="actionEdit_Settings">
   <property name="text">
    <string>Change Settings</string>
   </property>
  </action>
  <action name="actionShow_Code">
   <property name="text">
    <string>Show rules (prolog code)</string>
   </property>
  </action>
  <action name="actionShow_information">
   <property name="text">
    <string>Show information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>ElementView</class>
   <extends>QTreeView</extends>
   <header>view/elementview.hpp</header>
  </customwidget>
  <customwidget>
   <class>PredefinedElementView</class>
   <extends>QListView</extends>
   <header>view/predefinedelementview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
